src/monopoly.js

```diff
@@ -27,7 +27,9 @@ var monopoly = new Game(players, board, turn);
 
 var turn = new Turn(monopoly);
 
-var view = new MonopolyView(monopoly, turn);
+monopoly.setTurn(turn);
```

Comment:
Set turnMap of the game after we create the turn. If I don't do this then the game can find the turnMap. Not sure if there is a better way of doing this.